packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -174,21 +176,23 @@ const Dropdown = ({
     handleBlur,
     has_symbol,
     hint,
-    label,
-    list,
-    name,
-    no_border,
+    initial_offset = 0,
     is_alignment_top,
     is_alignment_left,
     is_align_text_left,
     is_large,
+    is_mobile,
```

Comment:
We shouldn't need `is_mobile` here, `Dropdowns` aren't used in mobile, for touch and smaller screens we should use `SelectNative` instead.

Reply:
I removed the `is_mobile` and use  `<DesktopWrapper>`